tests/instrumentation/psycopg2_tests.py

```diff
@@ -14,7 +14,7 @@
 except ImportError:
     has_psycopg2 = False
 
-travis_and_psycopg2 = 'TRAVIS' not in os.environ or not has_psycopg2
+travis_and_psycopg2 = False # 'TRAVIS' not in os.environ or not has_psycopg2
```

Comment:
Should the comment be changed?

Reply:
I leave this up to @beniwohli if he generally wants to adapt this test handling.

Reply:
we could change it to

    has_postgres_configured = 'POSTGRES_DB' in os.environ

and then change the skips below to

    @pytest.mark.skipif(not has_postgres_configured, "PostgresSQL not configured")

Reply:
will change